docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
 ////
-This guide is maintained in the main Quarkus repository
+This guide is maintained in the main {ProductName} repository
```

Comment:
this seems wrong. it is always in the man Quarkus repository - does not change based product name.

Reply:
Reverted back to Quarkus because this comment will not be included in product documentation. However, you should know that we always use variables for product names so that if the name changes it is an easy update. Also, if the name differs between community and enterprise, which at the moment, the short form of the name does not.